tests/onnx_backend_test.py

```diff
@@ -3,14 +3,23 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import unittest
+import os
 
+import unittest
 import onnx.backend.test
 
 import onnx_caffe2.backend as c2
 
+# This is a pytest magic variable to load extra plugins
+pytest_plugins = 'onnx.backend.test.report',
+
+backend_test = onnx.backend.test.BackendTest(c2, __name__)
+if 'TRAVIS' in os.environ:
```

Comment:
do we want to exclude it only for c2 backend or for everything?

Reply:
IMO only c2, potentially other backends could have more memory efficient solution to support running it on travis ;)